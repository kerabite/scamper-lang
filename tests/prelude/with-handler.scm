(with-handler
  (lambda (err) (string-append "This is the error that was generated: " err))
  (lambda (x y z) (+ x y z))
  1 2 3)

(with-handler
  (lambda (err) (string-append "This is the error that was generated: " err))
  (lambda (x y z) (error "oh no, an error!"))
  1 2 3)
