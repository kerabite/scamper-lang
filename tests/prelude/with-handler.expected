6
"This is the error that was generated: A runtime error was encountered: oh no, an error!"
